README.md

```diff
@@ -256,13 +277,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Check out the [Globo website](https://hacktoberfest.globo.com/) for more details.
 - **Notes**: Only the first 100 people residing in Brazil will receive the t-shirt.
 
-#### **Earthly** (Stickers)
+#### **Milvus** (sticker pack and digital badge)
 
-- **Requirements**: Contribute code to the [Earthly](https://github.com/earthly/earthly) open source build tool or contribute an Earthfile build to any public repo.
-- **Swag**: Stickers
-- **How to sign up**: Complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8). All PRs that are contributing value will get an Earthly sticker!
-- **Issues**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest)
-- **Notes**: If you need any assistance, [join their Slack channel](https://earthly.dev/slack). See their [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+- **Requirements**: Contribute to at least one merged pull request.
+- **Swag**: a sticker pack & a digital Milvus contributor badge.
+- **How to sign up**: Sign up and get more info [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating a [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues tagged for Hacktoberfest are available on the [Milvus Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9) and view the [Hacktoberfest thread here.](https://discuss.milvus.io/t/join-hacktoberfest-2021-with-us/72)
```

Comment:
I formatted this to be more in line with the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing). Using this as a template, please edit the other entries below.